suite/auto-sync/CppTranslator/arch_config.json

```diff
@@ -28,91 +28,91 @@
   "ARM": {
     "files_to_translate": [
       {
-        "in": "{AUTO_SYNC_ROOT}/vendor/llvm_root/llvm/lib/Target/ARM/Disassembler/ARMDisassembler.cpp",
```

Comment:
It was removed because I updated the auto-sync root path passed to the translator. I noticed that all the files accessed from the updater script and the translator/differ etc. are inside the llvm folder so `{AUTO_SYNC_ROOT}` is a path to the folder (instead of the root directory of llvm-capstone). This wasn't really necessary but I did it to simplify the paths. I should have left a hint for that.

Reply:
Ah ok. Guess it is fine then.